### PR TITLE
fix: [CDS-86708]: Add Scoped Service Url in Pipeline

### PR DESCRIPTION
**Pull request by pankajsingh-harness**
### Summary

**Issue:** Wrong service URL getting created for org and account level service in Service tab in Pipeline

**Fix:** Used scoped URL so that the correct URL gets generated for the org and account service.

#### Screenshots
**Before:** Wrong service URL getting generated so no service data getting populated in the service page.

https://github.com/harness/harness-core-ui/assets/110591046/c63d43d5-4a21-4139-8cc2-828bd5acd4b6

**After:** Correct URL getting generated and service data getting populated properly

https://github.com/harness/harness-core-ui/assets/110591046/253a07fc-3069-4de9-a8e8-404d6ea02a62

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
- MessageMetadata: `trigger messagecheck`
- GitLeaks: `trigger gitLeaks`
</details>

## PR check failures and solutions
https://harness.atlassian.net/wiki/spaces/BT/pages/21106884744/PR+Checks+-+Failures+and+Solutions

### Rebase Instructions:
To rebase your branch onto the latest changes in the target branch, simply use the following command in a comment: `/rebase`

### CODE REVIEW

File: src/modules/75-cd/components/PipelineSteps/DeployServiceEntityStep/__tests__/servicesMetadata.json [hunk 1]
```diff
@@ -1,12 +1,5 @@
 {
   "serviceV2YamlMetadataList": [
-    {
```
Comment (pankajsingh-harness) [inline src/modules/75-cd/components/PipelineSteps/DeployServiceEntityStep/__tests__/servicesMetadata.json:3]: Removed this to have one-to-one mapping with all service data.

File: src/modules/75-cd/components/PipelineSteps/DeployServiceEntityStep/__tests__/services.json [hunk 2]
```diff
@@ -10,11 +10,5 @@
       "identifier": "svc_2",
       "name": "Service 2"
     }
-  },
```
Comment (pankajsingh-harness) [inline src/modules/75-cd/components/PipelineSteps/DeployServiceEntityStep/__tests__/services.json:13]: Removed this to have one-to-one mapping with all service data.

File: src/modules/75-cd/components/PipelineSteps/DeployServiceEntityStep/__tests__/__snapshots__/DeployServiceEntityWidget.SingleService.test.tsx.snap [hunk 3]
```diff
@@ -238,7 +238,8 @@ exports[`DeployServiceEntityWidget - single service tests user can delete select
                       class="serviceNameWrapper"
                     >
                       <a
-                        href="/account/undefined/services/svc_3"
```
Comment (pankajsingh-harness) [inline src/modules/75-cd/components/PipelineSteps/DeployServiceEntityStep/__tests__/__snapshots__/DeployServiceEntityWidget.SingleService.test.tsx.snap:241]: Updated snapshot as of now. We will remove the snapshot once look again for the test update.

File: src/modules/75-cd/components/PipelineSteps/DeployServiceEntityStep/__tests__/__snapshots__/DeployServiceEntityWidget.MultiService.test.tsx.snap [hunk 4]
```diff
@@ -242,141 +242,6 @@ exports[`DeployServiceEntityWidget - multi services tests user can delete select
           data-rbd-droppable-context-id="1"
           data-rbd-droppable-id="serviceDropper"
         >
-          <div
```
Comment (pankajsingh-harness) [inline src/modules/75-cd/components/PipelineSteps/DeployServiceEntityStep/__tests__/__snapshots__/DeployServiceEntityWidget.MultiService.test.tsx.snap:245]: Updated snapshot as of now. We will remove the snapshot once look again for the test update.

**Comment by pankajsingh-harness** [comment]
trigger messagecheck

File: src/modules/75-cd/components/PipelineSteps/DeployServiceEntityStep/__tests__/servicesMetadata.json (hunk 1 above)
Comment (ishantmehtaharness) [reply src/modules/75-cd/components/PipelineSteps/DeployServiceEntityStep/__tests__/servicesMetadata.json:3]: cool

**Review by ishantmehtaharness — APPROVED** [review]
looks good

**Review by chetanvekariya-harness — APPROVED** [review]

**Review by samarth-sharma31 — APPROVED** [review]

**Pull request merged**